### PR TITLE
- adds missing service limits information for ODSP and planner

### PR DESCRIPTION
**Pull request by baywet**

### CODE REVIEW

File: concepts/throttling.md [hunk 1]
```diff
@@ -278,6 +278,20 @@ The following limits apply to any request on `/security`.
 The preceding limits apply to the following resources: 
 openTypeExtension, schemaExtension, administrativeUnit, contact, device, event, group, message, organization, post, and user.
 
+### OneDrive and SharePoint service limits
+
+Service limits for OneDrive, OneDrive for Business and SharePoint Online are not available. For more information, see [why can't you just tell me the exact throttling limits?](/sharepoint/dev/general-development/how-to-avoid-getting-throttled-or-blocked-in-sharepoint-online#why-cant-you-just-tell-me-the-exact-throttling-limits) from SharePoint's documentation.
```
Comment (davidmu1) [inline concepts/throttling.md:283]: Service limits for OneDrive, OneDrive for Business, and SharePoint Online are not available.

Comment (baywet) [reply concepts/throttling.md:283]:
```suggestion
Service limits for OneDrive, OneDrive for Business, and SharePoint Online are not available. For more information, see [why can't you just tell me the exact throttling limits?](/sharepoint/dev/general-development/how-to-avoid-getting-throttled-or-blocked-in-sharepoint-online#why-cant-you-just-tell-me-the-exact-throttling-limits)
```

File: concepts/throttling.md [hunk 2]
```diff
@@ -278,6 +278,20 @@ The following limits apply to any request on `/security`.
 The preceding limits apply to the following resources: 
 openTypeExtension, schemaExtension, administrativeUnit, contact, device, event, group, message, organization, post, and user.
 
+### OneDrive and SharePoint service limits
```
Comment (Lauragra) [inline concepts/throttling.md:281]: Global comments: Should we alphabetize the H3s under Service-specific limits? I don't know that the H3s follow a specific order. Also, should the H3s map to the IA, i.e. This heading would be "Files and sites and lists" and the next would be "Tasks and plans"?

Comment (baywet) [reply concepts/throttling.md:281]:
```suggestion
### Files and lists service limits
```

File: concepts/throttling.md [hunk 3]
```diff
@@ -278,6 +278,20 @@ The following limits apply to any request on `/security`.
...
+
+Service limits for OneDrive, OneDrive for Business, and SharePoint Online are not available. For more information, see [why can't you just tell me the exact throttling limits?](/sharepoint/dev/general-development/how-to-avoid-getting-throttled-or-blocked-in-sharepoint-online#why-cant-you-just-tell-me-the-exact-throttling-limits).
+
+The preceding information applies to the following resources:  
+baseItem, baseItemVersion, columnDefinition, columnLink, contentType, drive, driveItem, driveItemVersion, fieldValueSet, itemActivity, itemActivityStat, itemAnalytics, list, listItem, listItemVersion, permission, sharedDriveItem, site, and thumbnailSet.
+
+### Planner service limits
```
Comment (baywet) [inline concepts/throttling.md:288]:
```suggestion
### Tasks and plans service limits
```

**Comment by baywet** [comment]
@Lauragra to address the second part of your comment:
I'm planning to split the service limits in a separate page as a recommendation that @angelgolfer-ms made a while ago, do a little of cleanup (this include sorting the titles), promote H3's to H2's so we can right side navigation...
But before that I need to wait for my last PR to be merged to avoid generating conflicts https://github.com/microsoftgraph/microsoft-graph-docs/pull/8451
This PR is in turns pending on service deployments of the documented limits, which I expect to happen in August.

Hopefully that helps provide additional context. This PR should be good to merge at this time. :)

**Pull request merged**